source/test/java/com/restfb/types/PostTest.java

```diff
@@ -294,4 +297,30 @@ public void checkV2_8_feedTargetingScheduleUntil() {
     Date ts = examplePost.getFeedTargeting().getRelevantUntilTs();
     assertEquals(1457620560000L, ts.getTime());
   }
+
+  @Test
+  public void checkV2_5_feedTargetingWithRegions() {
+    Post examplePost = createJsonMapper().toJavaObject(jsonFromClasspath("v2_5/post-with-feedtargeting-regions"), Post.class);
```

Comment:
this test currently fails to deserialize because the format of the region in the geo_locations is inconsistent between versions of responses.